crates/lox_core/src/bodies.rs

```diff
@@ -62,8 +64,223 @@ pub fn gravitational_parameter<T: PointMass>(_: T) -> f64 {
     <T as PointMass>::gravitational_parameter()
 }
 
+const N_COEFFICIENTS: usize = 18;
+
+type PolynomialCoefficients = (f64, f64, f64, [f64; N_COEFFICIENTS]);
+type NutationPrecessionCoefficients = ([f64; N_COEFFICIENTS], [f64; N_COEFFICIENTS]);
+
+pub trait RotationalElements {
```

Comment:
If possible, it would be great to be in the habit of leaving doc comments that explain what something is for/does in simple terms that a non-astrodynamicist would grasp its use. Future software oompa loompas will thank you!

E.g. "A body implementing RotationalElements specifies the coefficients necessary to calculate X, Y and Z, default implementations of which are provided."

Reply:
Guilty as charged. Will add docs in a subsequent PR.